e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/find-utilisation-reports-by-bank-and-year.spec.js

```diff
@@ -85,14 +85,16 @@ context('PDC_RECONCILE users can search for reports by bank and year', () => {
     pages.searchUtilisationReportsResultsPage.noReportsText().should('exist');
   });
 
-  it('should display the dropdown for banks which have reports and should not display anything for banks without reports', () => {
+  it('should disable the input until a bank is selected and display the dropdown for banks which have reports and should not display anything for banks without reports', () => {
```

Comment:
Could we split this test up? Lots of "ands" in 1x test, thinking something like this?:

```js
- when a bank is not selected
  - it should disable the input

- when a bank is selected
  - it should display the dropdown for banks which have reports
  - it should not display anything for banks without reports
```

Reply:
Decided to just add a new test for the disable/enable functionality as I'd like to leave the datalist selection all inside of one test